### DISCUSSION TITLE
leafmap overture javascript error in Visual code studio

### DISCUSSION THREAD

**Discussion by SHsabbir25** [giswqs/geog-312#197] [Q&A] [answered]
![Screenshot 2024-12-20 212957](https://github.com/user-attachments/assets/69f65930-051f-474f-9639-3b59f43cb381)
![Screenshot 2024-12-20 213300](https://github.com/user-attachments/assets/536b6f35-3494-436f-bac8-3a74accc1d68)

**Comment by giswqs** [comment, marked as answer]
Maplibre doesn't work in VS Code. Use JupyterLab instead
